# ML_kmerBin
EVALUATING LEARNING METHODS TO CLASSIFY DNA SEQUENCES IN METAGENOMICS
Metagenomics enables the study of bacteria from environmental samples using DNA sequences. An essential task in metagenomic experiments is to assign each sequence to a particular specie. Although several supervised and unsupervised methods have been proposed, no standard tool is available for this task. We analyzed Supervised Learning models (i.e., Random forest, K-nearest neighbors, and neuronal networks) to bin DNA sequences for metagenomic experiments and evaluated them using the CAMI benchmark datasets. The results indicate that the models can rightly assign most contigs to a particular species for the most abundant species in the metagenome. However, their performance decreases for less representative organisms. Codes and Supplementary Material are available from the GitHub: http/github/andvides/ML_kmerBin. 
